### DISCUSSION TITLE
How to use grpcurl to test gRPC servers

### DISCUSSION THREAD

**Discussion by giscus** [appscode/community#59] [Blog]
# How to use grpcurl to test gRPC servers

AppsCode's latest news and announcements, tips and tricks for our cloud-native ecosystem and products such as KubeDB, Stash, Voyager, ByteBuilders, KubeVault, and many more.

https://byte.builders/blog/post/grpcurl/

**Comment by wmstantoniii** [comment]
Simple and to the point!  Thanks.
